### PR TITLE
Deprecations, disables, deletions for 3.4.0

### PR DESCRIPTION
**Pull request by MikeMcQuaid**
Do not merge until the next tag will be 3.4.0.

### CODE REVIEW

File: Library/Homebrew/dev-cmd/pr-upload.rb [hunk 1]
```diff
@@ -90,7 +90,7 @@ def bottles_hash_from_json_files(json_files, args)
   def pr_upload
     args = pr_upload_args.parse
 
-    # odeprecated "`brew pr-upload --github-org`", "`brew pr-upload` without `--github-org`" if args.github_org
+    odisabled "`brew pr-upload --github-org`", "`brew pr-upload` without `--github-org`" if args.github_org
```
Comment (Bo98) [inline Library/Homebrew/dev-cmd/pr-upload.rb:93]: Use the `replacement` feature of the CLI parser instead so it gets hidden from documentation.

(I'm assuming it's intention to go directly to `odisabled` here.)

Comment (MikeMcQuaid) [reply Library/Homebrew/dev-cmd/pr-upload.rb]: @Bo98 Already hidden but not intended to go straight to `odisabled`, fixed, thanks.

Comment (Bo98) [reply Library/Homebrew/dev-cmd/pr-upload.rb]: You also can go and remove all the existing args that have `replacement:`.

Comment (MikeMcQuaid) [reply Library/Homebrew/dev-cmd/pr-upload.rb]: @Bo98 Good shout, done and added to docs too.

File: docs/Releases.md [hunk 2]
```diff
@@ -23,7 +23,8 @@ Homebrew release:
 
 If this is a major or minor release (e.g. X.0.0 or X.Y.0) then there are a few more steps:
 
-1. Before creating the tag you should delete any `odisabled` code, make any
+1. Before creating the tag you should delete any `odisabled` code or
+   `replacement:` `switch` or `flag`, make any
    `odeprecated` code `odisabled`, uncomment any `# odeprecated` code and add
    any new `odeprecations` that are desired. Also delete any command argument
    definitions that pass `replacement: ...`.
```
[the remark this answers is not in the document]
Comment (MikeMcQuaid) [reply docs/Releases.md]: @Bo98 Just removed. Cannot read today apparently 🙃

**Pull request merged**